ReadBackwards.pm

```diff
@@ -9,11 +9,12 @@ use strict ;
 
 use vars qw( $VERSION ) ;
 
-$VERSION = '1.05' ;
+$VERSION = '1.06_01' ;
 
 use Symbol ;
 use Fcntl qw( :seek O_RDONLY ) ;
 use Carp ;
+use Scalar::Util 'openhandle';
```

Comment:
This was added to core in 5.7.3, so we would either have to drop 5.6 support or add a prereq (currently this module doesn't have any non-core deps).  Pretty sure that doesn't matter, but something to consider.

Reply:
Current Scalar::Util is broken on Perls older that 5.8.6.  The ticket has (probably correctly) so far been ignored:
https://rt.cpan.org/Public/Bug/Display.html?id=132365
so this would in effect be dropping support for older Perls.